Dockerfile.apidocs

```diff
@@ -0,0 +1,51 @@
+# Sentry api documentation generation environment
+#
+# Instructions:
+#
+#   Build the container:
```

Comment:
We should wrap all of this into a simple that thing is just `bin/update-api-docs` or whatever, that does these docker commands and mounts the volume, etc.

Reply:
Ok.